WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -309,9 +299,9 @@ <h2 class="text-right text-lowercase">Available for <br /><span itemprop="operat
                                         <small class="text-muted float-left">.dmg</small>
                                         <small class="float-right">
                                             <!-- macOS Signature Link -->
-                                            <a href="https://github.com/zkSNACKs/WalletWasabi/releases/download/v1.1.12/Wasabi-1.1.12.dmg.asc" class="text-uppercase" rel="external nofollow">Signature</a>&nbsp;/
+                                            <a href="https://github.com/zkSNACKs/WalletWasabi/releases/download/v1.1.12/Wasabi-1.1.12.dmg.asc" title="Download Wasabi Wallet's macOS signature" class="text-uppercase" rel="external nofollow">Signature</a>&nbsp;/
```

Comment:
Maybe we should say: `Download PGP signature for macOS`

Reply:
```suggestion
                                            <a href="https://github.com/zkSNACKs/WalletWasabi/releases/download/v1.1.12/Wasabi-1.1.12.dmg.asc" class="text-uppercase" rel="external nofollow">Signature</a>&nbsp;/
```